### DISCUSSION TITLE
Does OrcaSlicer slow down your PC (W11)?

### DISCUSSION THREAD

**Discussion by joaopjrn** [OrcaSlicer/OrcaSlicer#4315] [Q&A]
So, like the title says, does it slow down your PC as a whole?
I cant keep orca open on my computer, because it slows down the whole OS, literally unusable until i close it.

Here is what i have:
Windows 11 PRO (23H2 - 22631.3235)
32GB DDR4
RTX 4070
Ryzen 7 3700X

**Comment by tsmith35** [comment]
I haven't experienced any such slowdowns due to OrcaSlicer. Your system looks like it's more than capable, so you will need to do some more digging. I'd start by opening OrcaSlicer, then while it's running, hit Ctrl+Shift+Esc to open task manager. From there, go to the processes tab and click the column header for CPU to sort by CPU usage. That's usually the quickest/easiest way to check.

**Comment by AlatusAles** [comment]
Подтверждаю. Даже при бездействии ORCA и/или свернутом окне потребление ресурсов ПК очень большое! 
![new-1](https://github.com/SoftFever/OrcaSlicer/assets/143936089/459d4ec0-1f9d-4613-96ed-5c6378af81df)
![new-2](https://github.com/SoftFever/OrcaSlicer/assets/143936089/1e13c850-241f-4224-836b-61e1b70e40af)

**Comment by vgdh** [comment]
Try latest dev 2.1 it's much lighter with resource consumption for my experience.

**Reply by AlatusAles** [reply]
Меньше, но по моему мнению все еще много для неактивного окна программы не выполняющей никаких расчетов в реальном времени (40% CPU и 70% GPU...).
![new-2 1](https://github.com/SoftFever/OrcaSlicer/assets/143936089/2777440d-f014-44f1-83cf-fe060fa03d41)

**Comment by FlowForm** [comment]
Yes definitely having problems with this and even on a 7950x with 96gb ram.! When the "Device" tab is open, this is pretty bad and affects the GUI of my 3DS Max a lot. This is not acceptable.

**Comment by tsmith35** [comment]
What CPU, GPU and RAM do each of you have? Running with an SSD or HDD?

**Reply by AlatusAles** [reply]
Ryzen 7 5700X
Radeon RX 6650 XT
32 Gb (3200 MHz)
SATA SSD (only for programs)
.....
На другом ПК аналогичная ситуация с неоправданным использованием ресурсов.

**Reply by AlatusAles** [reply]
Дополню. Данная проблема появляется только в режиме "Предпросмотр нарезки".
![new-e](https://github.com/SoftFever/OrcaSlicer/assets/143936089/d899e6e0-ba3e-4fdf-b688-99535a4ef2ed)
![new-r](https://github.com/SoftFever/OrcaSlicer/assets/143936089/9761635e-3e06-4e31-84ed-81ef6993c955)

**Comment by LJKraus** [comment]
I have never seen any slowdown from Orcaslicer. 

Win 11 Pro, i7-8700K CPU @ 3.70GHz, 32GB DDR4, Nvidia GTX1070 8GB
Capable, but wasn't even state of the art when purchased six years ago.

Two instances of OrcaSlicer 2.0.0-beta, one with four plates of large models, one with one plate, device videos in each:   9% CPU, <10% memory.
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/03a030d3-73fd-494d-9192-56b5c26b1911)


I'd look for sub-optimal configurations in the system.

**Reply by AlatusAles** [reply]
https://github.com/SoftFever/OrcaSlicer/discussions/4315#discussioncomment-9130113

и ответ ниже.

**Comment by AlatusAles** [comment]
I managed to get closer to the problem! Large resource consumption occurs with some translation localizations. For example, with English and Italian languages everything is ok, but with Ukrainian and Russian this problem arises. Perhaps this will help professional developers.
![en](https://github.com/SoftFever/OrcaSlicer/assets/143936089/eef67490-2235-4009-99eb-7f97b409ee46)
![it](https://github.com/SoftFever/OrcaSlicer/assets/143936089/e865751c-a1bc-4514-b7d6-23ec9465ed9b)
![ru](https://github.com/SoftFever/OrcaSlicer/assets/143936089/7230e540-f68e-469e-b80a-7e71da09af0d)
![uk](https://github.com/SoftFever/OrcaSlicer/assets/143936089/da169254-aff8-466c-81a4-132bc7a1c3d5)

**Comment by tsmith35** [comment]
So I am running v2.1.0-dev from the 4-15-24 build. Win 10 x64 with EN-US localization. I went to Preferences and went through all languages from start to finish. No unusual loading occurred, with CPU usage typically at 0% to 0.1% after the program loaded. Perhaps both the PC and slicer must be on the same localization for the issue to occur.

BTW, I see that PR #4981 was pushed through 3 days ago with the comment, "Russian translation update OrcaSlicer V2.0.0 Official Release. Improved translation, corrected errors."

**Reply by AlatusAles** [reply]
2.1.0 - Проблема осталась.
https://github.com/SoftFever/OrcaSlicer/discussions/4315#discussioncomment-9127429
Позже, переключу локализацию своей Win 10_x64 на EN-US и протестирую.
Хотя, лично для меня эта проблема не существенна (нет смысла часами смотреть на "Предпросмотр"), но возможно вам, как разработчику, это подскажет что-то про оптимизацию. 
В любом случае, Спасибо, всей команде, за внимание и этот Прекрасный Слайсер!

**Comment by J3LS** [comment]
Possible connection to this issue.

Background
I've just swapped ISP so changed router. Even though Orca has the new credentials I can't connect so back to a flash drive again. I can live with that until I solve it but...

Until now (with a good connection to my Ender 3 V3 KE) just leaving Orca running but not being used for a few minutes resulted in high cpu usage and the fan going bananas. Now I have the "Could not connect: No route to host" on the device page but leaving Orca running for any amount of time does nothing. CPU etc. remain idle and all quiet.

Sorry, I can't verify this by connecting again! (For the moment).
